contracts/ownership/Superuser.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.4.23;
+
+
+import "./Ownable.sol";
+import "./rbac/RBAC.sol";
+
+
+/**
+ * @title Superuser
+ * @dev The Superuser contract defines a single superuser who can transfer the ownership 
+ * @dev of a contract to a new address, even if he is not the owner. 
+ * @dev A superuser can transfer his role to a new address. 
+ */
+contract Superuser is Ownable, RBAC {
+  event SuperuserTransferred(address indexed previousSuperuser, address indexed newSuperuser);
```

Comment:
These events aren't necessary. RBAC emits `RoleChanged` events, so `SuperuserTransferred` isn't needed, and Ownable already emits `OwnershipTransferred`.

Reply:
About OwnershipTransferred: I am overriding transferOwnership and therefore it will not be emitted by Ownable. 

I need to override it because in Ownable that method is onlyOwner and I need it to be onlySuperuser (actually it should be both).

I think I could do "emit super.OwnershipTransferred..." though.

Reply:
1) the events are inherited as well, so you don't have to declare it again. just `emit OwnershipTransferred(...)`
2) If you call `super.transferOwnership(...)` the contract would emit the event correctly regardless

Reply:
@shrugs I removed the event declaration. As we discussed in Slack, I can't call super.transferOwnership because of the onlyOwner modifier.